src/dev/src/app/nav/navs.demo.scss

```diff
@@ -1,5 +1,3 @@
 // Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
```

Comment:
Is there a compelling reason to keep this file?

Reply:
Nope